util/deadlockhistory/deadlock_history.go

```diff
@@ -148,12 +148,13 @@ func (d *DeadlockHistory) GetAllDatum() [][]types.Datum {
 				row[5] = strings.ToUpper(hex.EncodeToString(item.Key))
 			}
 
-			row[6] = nil
-			if item.AllSQLDigests != nil {
-				row[6] = "[" + strings.Join(item.AllSQLDigests, ", ") + "]"
-			}
+			row[6] = item.TxnHoldingLock
 
-			row[7] = item.TxnHoldingLock
+			// TODO: Implement the ALL_SQL_DIGESTS column for the deadlock table.
+			//row[7] = nil
+			//if item.AllSQLDigests != nil {
+			//	row[7] = "[" + strings.Join(item.AllSQLDigests, ", ") + "]"
+			//}
```

Comment:
I prefer to delete them and comment out the column definition. 😕

Reply:
Updated. Is this what you mean?